test/test.js

```diff
@@ -262,7 +271,8 @@ describe('Pieces Exporter', function () {
     jobInfo = await apos.http.post('/api/v1/article/export?apikey=testKey', {
       body: {
         extension: 'csv',
-        batchSize: 10
+        batchSize: 10,
```

Comment:
We were going to move this to a module configuration parameter so you can still use it for unit tests but the actual client can't specify problematic values.

Reply:
That was for expiration.

Reply:
https://github.com/apostrophecms/piece-type-exporter/pull/1#discussion_r733781795